specs/validations.md

```diff
@@ -156,14 +170,99 @@ interface L2OutputOracle {
 }
 ```
 
+### Validation Rewards
```

Comment:
Why this section is duplicated with the section in `## Validator Pool Smart Contract`? I think this section would be better to be removed.